test(jans-auth-server): adapt test after new methods added into SessionIdService

**Describe the bug**

After changes in `SessionIdService` service test was not updated and is failing.

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:4.0.0-M7:site (default-site) on project jans-auth-server-parent: Failed to get report for org.apache.maven.plugins:maven-javadoc-plugin: Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:testCompile (default-testCompile) on project jans-auth-server: Compilation failure
[ERROR] /home/jetty/.jenkins/workspace/jans-auth-server/jans-auth-server/server/src/test/java/io/jans/as/server/service/SessionIdServiceTest.java:[87,37] reference to hasAllScopes is ambiguous
[ERROR]   both method hasAllScopes(io.jans.as.common.model.session.SessionId,java.util.Set<java.lang.String>) in io.jans.as.server.service.SessionIdService and method hasAllScopes(java.lang.String,java.util.Set<java.lang.String>) in io.jans.as.server.service.SessionIdService match
[ERROR] 
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

https://jenkins.jans.io/jenkins/job/jans-auth-server/9981/console

--- a/jans-auth-server/server/src/test/java/io/jans/as/server/service/SessionIdServiceTest.java
+++ b/jans-auth-server/server/src/test/java/io/jans/as/server/service/SessionIdServiceTest.java
@@ -84,7 +84,7 @@ public class SessionIdServiceTest {
 
     @Test
     public void hasAllScopes_whenSessionIsNull_shouldReturnFalse() {
-        assertFalse(sessionIdService.hasAllScopes(null, null));
+        assertFalse(sessionIdService.hasAllScopes((SessionId) null, null));
     }
 
     @Test

